src/main/java/com/payfurl/payfurlsdk/models/PayIdDetails.java

```diff
@@ -0,0 +1,47 @@
+package com.payfurl.payfurlsdk.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Date;
+
+public class PayIdDetails {
+    private final String payId;
+    private final String payIdType;
+
+    @JsonCreator
+    public PayIdDetails(@JsonProperty("PayId") String payId,
+                        @JsonProperty("PayIdType") String payIdType) {
+        this.payId = payId;
+        this.payIdType = payIdType;
+    }
+
+    public String getPayId() {
+        return payId;
+    }
+    public String getPayIdType() {
+        return payIdType;
+    }
+
+
+    public static class Builder {
+        private String payId;
+        private String payIdType;
+
+
+        public Builder withPayId(String payId) {
+            this.payId = payId;
+            return this;
+        }
+
+        public Builder withPayIdType(String payIdType) {
+            this.payIdType = payIdType;
+            return this;
+        }
+
```

Comment:
Please remove empty line

Reply:
Also, you could use formatting that we usually use 
![image](https://user-images.githubusercontent.com/113937834/221785757-445c6601-3405-44a7-840d-20c5e580e1ae.png)